### ISSUE TITLE
[feature request] Add option to also fzf the Description of Packages

### ISSUE THREAD

**Issue by tracure1337** [ericlay/fuzzy-pkg-finder#5]
[feature request] Add option to also fzf the Description of Packages.

This would be so helpful!

**Comment by ericlay** [comment]
This would be very useful. Right now the script just does some sorting from pacman results and then the preview window uses pacman files database commands for the highlighted line or package. 

There would need to be a way to include the description with the package name as fzf itself simply presents, searches/matches and selects list items for hand off. An ugly option would be to zipper the description into the package list. 

It's def worth looking into.

**Comment by ericlay** [comment]
So I've solved the flow of it and am working it out in psuedo code. I've created a dev branch to actually totally rebase a lot of the script using gawk, adding description searching to aur as well. In the mean time, if you swap line 58 (or entire file) as seen in [this commit ](https://github.com/ericlay/fuzzy-pkg-finder/compare/master...dev#diff-42db0bfebfce2fece1e79a55d7d6e87ea14dcc24dcd26678f4b7b854c8884fce), the standard repo's will have search-able descriptions!
